README.md

```diff
@@ -62,6 +62,8 @@ Visit the Pi's IP address (e.g. http://192.168.1.10/) and use the `pihole_passwo
 
 Visit the Pi's IP address with port 3030 (e.g. http://192.168.1.10:3030/), and log in with username `admin` and the password `monitoring_grafana_admin_password` you configured in your `config.yml`.
 
+Go to [http://localhost:3030/d/o9mIe_Aik/internet-connection](http://localhost:3030/d/o9mIe_Aik/internet-connection) (change `localhost` to your docker host ip/name).
```

Comment:
The line above explains how to get to the dashboards, though—if anything, we would just want to add in directions for how to click through Grafana's UI to get to the full dashboards listing so you can choose which dashboard you want to visit in there.

Reply:
Sure, I can change that wording. Is that the direction we want to go? There were a couple other ideas in #309